app/models/job.rb

```diff
@@ -19,4 +20,8 @@ def assign_tech
     tech_matches = Technology.where(name: raw_technologies)
     self.technologies = tech_matches
   end
+
+  def self.by_tech(tech_name)
+    joins(:technologies).where(technologies: {name: tech_name})
```

Comment:
so I came across this weird note that Rails "officially" prefers class methods if you pass in a param so I shrugged and made a class method instead of a scope. But it does make me feel weird. ;)